content/docs/5.0/get-started/tikv-in-5-minutes.md

```diff
@@ -0,0 +1,209 @@
+---
+title: TiKV in 5 Minutes
+description: Get started using TiKV in 5 Minutes
+menu:
+    "5.0":
+        parent: Get Started
+        weight: 1
+---
+
+## Prerequisites
+
+Ensure that you are using MacOS or Linux.
+
+{{< warning >}}
+The following steps are only for test environments.
+{{< /warning >}}
+<!-- TODO: For production environments please refer to ??? -->
+
+## Step1: Install a Playground TiKV Cluster
+
+1. Install TiUP by executing the following command:
+```bash
+$ curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+```
+
+2. Set the TiUP environment variables:
+
+Redeclare the global environment variables:
+
+```bash
+$ source .bash_profile
+```
+
+Confirm whether TiUP is installed:
+```bash
+$ which tiup
+```
+
+3. If TiUP is already installed, update the TiUP playground component to the latest version:
+```bash
+$ tiup update --self && tiup update playground
+```
+
+4. Use TiUP playground to start a local TiKV cluster
+```bash
+$ tiup playground --mode tikv-slim
```

Comment:
Even I double-ran Step 3, I ran into:

```
Starting component `playground`: /home/andylokandy/.tiup/components/playground/v1.4.3/tiup-playground --mode tikv-slim
Error: unknown flag: --mode
Error: run `/home/andylokandy/.tiup/components/playground/v1.4.3/tiup-playground` (wd:/home/andylokandy/.tiup/data/SYPvINB) failed: exit status 1
```

Reply:
cc @iosmanthus

Reply:
i guess the PR is not released.

Reply:
I guess the PR is not released!

Reply:
Will be released in tiup v1.5.0
https://github.com/pingcap/tiup/pull/1365